app/views/partials/foyer/recap-situation.html

```diff
@@ -21,33 +21,21 @@ <h4 class="section-header">Vos ressources</h4>
     close-others="true"
     class="accordion-ressources">
     <accordion-group
-      ng-repeat="ressource in ressourcesNonTns"
+      ng-repeat="ressource in ressources"
       class="recap-{{ ressource.type.category }}"
       heading="{{ ressource.type.label }}"
       is-open="ressource.isOpen">
-      <div class="row" ng-repeat="month in months">
-        <div class="col-sm-8">
-          {{ month.label | uppercaseFirst }}
-        </div>
-        <div class="col-sm-4">
-          {{ ressource.totalMensuel[$index] }} €
+      <div ng-if="ressource.totalMensuel">
+        <div class="row" ng-repeat="month in months">
+          <div class="col-sm-8">
+            {{ month.label | uppercaseFirst }}
+          </div>
+          <div class="col-sm-4">
+            {{ ressource.totalMensuel[$index] }} €
```

Comment:
Mmh. Am I supposed to see an amount?
![capture d ecran 2015-10-13 a 09 29 02](https://cloud.githubusercontent.com/assets/222463/10448390/e08af30c-718c-11e5-98df-f7b870ffe3e2.png)